imdb.py

```diff
@@ -0,0 +1,99 @@
+import getpass
+import os
+import sys
+
+
+def count_elements_in_list(lst):
+    elements_dict = {}
+    for el in lst:
+        elements_dict[el] = elements_dict.get(el, 0) + 1
+    return elements_dict
+
+
+def nice_hat_for_years_file(years_file, counter=True):
+    years_file.write('Year | Amount  |' + counter * 'A' + '\n')
+    years_file.write('_____|_________|' + counter * '_' + '\n')
+
+
+def create_histogram(lst, filename='histogram.txt', vertical=True,
```

Comment:
this functions looks to big it would be better to split it into two (create vertical and usual) and refactor a bit